lib/Pipedrive.js

```diff
@@ -28,6 +30,77 @@
 			callback(error, collectionItems, additionalData);
 		}, false);
 	};
+	exports.OAuthAuthenticate = function (client_id, redirect_uri, res) {
+	    res.redirect('https://oauth.pipedrive.com/oauth/authorize?' +
+            qs.stringify({
+                client_id: client_id,
+                redirect_uri: redirect_uri,
+            }))
+	};
+
+	exports.OAuthAuthenticateCallback = function (req, res, redirect_uri, clientId, clientSecret) {
+	    var code = req.query.code || null;
+	    var authOptions = {
+	        url: 'https://oauth.pipedrive.com/oauth/token',
```

Comment:
Maybe that could be declared as `const` for more visibility

Reply:
it has node 0.10 as min support afaik